ignition/src/IgnitionEnvironment.cpp

```diff
@@ -73,32 +76,32 @@ gympp::gazebo::EnvironmentCallbacks* IgnitionEnvironment::Impl::envCallbacks()
 std::shared_ptr<ignition::gazebo::Server> IgnitionEnvironment::Impl::getServer()
 {
     // Lazy initialization of the server
-    if (!server) {
+    if (!gazebo.server) {
 
         assert(sdf.Element());
         assert(!sdf.Element()->ToString("").empty());
-        serverConfig.SetSdfString(sdf.Element()->ToString(""));
+        gazebo.config.SetSdfString(sdf.Element()->ToString(""));
 
         sdf::Root root;
         auto errors = root.LoadSdfString(sdf.Element()->ToString(""));
         assert(errors.empty()); // This should be already ok
 
         // Create the server
         gymppDebug << "Creating the server" << std::endl << std::flush;
-        serverConfig.SetUseLevels(false);
-        server = std::make_unique<ignition::gazebo::Server>(serverConfig);
-        assert(server);
+        gazebo.config.SetUseLevels(false);
+        gazebo.server = std::make_unique<ignition::gazebo::Server>(gazebo.config);
```

Comment:
There is a specific reason for using `std::make_unique` to create a `shared_ptr`?

Reply:
Yes: a mistake. Thanks! I didn't realize it. I suspect that in the beginning I was using an `unique_ptr` and then I switched to a `shared_ptr`.